README.md

```diff
@@ -83,6 +83,71 @@ jobs:
 | 1   | **exit_code**  | Integer       | The exit code indicating the success or failure of the GitHub Actions workflow. |
 | 2   | **status**     |'failure' \| 'success'| The status of the GitHub Actions workflow, indicating success or failure. |
 
+
+## CLI (Command Line Interface)
+
+### Help Example
+```shell
```

Comment:
issue: Let's avoid using the shell for the code block as it colors the output text in shell colors.
<img width="259" alt="Screenshot 2024-04-04 at 10 32 37 PM" src="https://github.com/opensource-nepal/commitlint/assets/35629644/b7042ce7-95a6-4a43-b000-550647fdabce">

Reply:
We can leave it blank.